### DISCUSSION TITLE
Upgrading rappasoft/laravel-livewire-tables from 0.3.3 to a higher version giving error

### DISCUSSION THREAD

**Discussion by gal01azabolcs** [rappasoft/laravel-livewire-tables#543] [Q&A]
Dear Rappasoft,

I want to upgrade laravel-livewire-tables, but when I'm upgrading to 1.0 it's giving error :Component AttributeBag::has does not exist. (View:.../resources/views/vendor/livewire-tables/bootstrap-4/components/table/row.blade.php)

My question is that how can I handle this?

Thx for this great componenet!

**Comment by fabio-ivona** [comment]
hi @gal01azabolcs which version of Laravel are you using?

**Reply by gal01azabolcs** [reply]
Hi @fabio-ivona , I'm using Laravel 8, with php 7.4

**Reply by gal01azabolcs** [reply]
This is the error: Method Illuminate|View|ComponentAttributeBag::has does not exist. (View:
/home/vagrant/code/ospatarul-dashboard/resources/views/vendor/livewire-tables/bootstrap-
4/components/table/row.blade.php)

**Reply by fabio-ivona** [reply]
it seems that your laravel version did not support `ComponentAttributeBag::has` and `class` methods (they have been introduced in laravel v8.27.0 i think)

yesterday I opened a PR #538 that, among other things, fixes this issue adding backward compatibility

we should just wait that @rappasoft reviews and merges it

**Reply by gal01azabolcs** [reply]
@fabio-ivona thanks for the quick response!!
